contrib/mixin/mixin.libsonnet

```diff
@@ -241,7 +241,7 @@
           {
             alert: 'etcdDatabaseHighFragmentationRatio',
             expr: |||
-              (last_over_time(etcd_mvcc_db_total_size_in_use_in_bytes[5m]) / last_over_time(etcd_mvcc_db_total_size_in_bytes[5m])) < 0.5
+              (last_over_time(etcd_mvcc_db_total_size_in_use_in_bytes[5m]) / last_over_time(etcd_mvcc_db_total_size_in_bytes[5m])) < 0.5 and etcd_mvcc_db_total_size_in_use_in_bytes > 104857600
```

Comment:
I am not a prometheus expert. Based on the official guide: https://prometheus.io/docs/prometheus/latest/querying/operators/#logical-set-binary-operators,  the operator `and` is an intersection operation on two vectors. But in this case, both side are scalars that is either 0 (false) or 1 (true).

It will be great if any prometheus expert can double confirm this.

Reply:
Alert is triggered if both are true. No gotchas.

Previous test failed because etcd_mvcc_db_total_size_in_use_in_bytes was below 100MB.
Current test succeeds because etcd_mvcc_db_total_size_in_use_in_bytes is above 100MB.

<img width="1511" alt="1" src="https://user-images.githubusercontent.com/595501/223647621-34c525a5-55b8-497d-9010-2d646e1a4fa4.png">
<img width="1512" alt="2" src="https://user-images.githubusercontent.com/595501/223647638-76b0ab48-4b46-42e8-aee4-9a236192d170.png">
<img width="1512" alt="3" src="https://user-images.githubusercontent.com/595501/223647655-7ed1bd79-7f56-4a98-9b15-0290387e1ceb.png">